rtorrent-flood/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '2'
+services:
+  plex:
+    container_name: rtorrent-flood
+    image: wonderfall/rtorrent-flood
+    restart: unless-stopped
+    ports:
+      - 49184:49184
+      - 6881:6881  # If DHT is enabled
+      - 8085:3000  # GUI
```

Comment:
I think it would be better to not give bad practises in the docker-compose sample.